test/fixtures/cookbooks/wrapper/Berksfile

```diff
@@ -0,0 +1,10 @@
+source 'https://supermarket.chef.io'
+
+group :integration do
+  cookbook 'apt'
+  cookbook 'yum'
+end
+
+cookbook 'elkstack', :path => '../../../../'
```

Comment:
Did you need to include a Berksfile here? I think if you put it in the main Berksfile, TK will pick it up.